vnext/Scripts/copyRNLibraries.js

```diff
@@ -14,16 +14,25 @@ const rnCopiesDir = path.join(
   path.dirname(require.resolve('react-native-windows/package.json')),
   'ReactCopies',
 );
+const rnTypesDir = path.dirname(
+  require.resolve('@types/react-native/package.json'),
+);
 
 exports.copyTask = baseDir => {
   const reactNative = (...files) => files.map(f => path.join(rnDir, f));
+  const reactNativeTypes = (...files) =>
+    files.map(f => path.join(rnTypesDir, f));
   const reactCopies = (...files) => files.map(f => path.join(rnCopiesDir, f));
   const src = (...files) => files.map(f => path.join(baseDir, 'src', f));
   const base = file => path.join(baseDir, file);
 
   return series(
     exports.cleanTask(baseDir),
 
+    copyTask({
+      paths: reactNativeTypes('*.d.ts'),
+      dest: base('src/rntypes'),
```

Comment:
do we have to copy it to `src` could we get away with in the `lib` or `dist` folders? If there is a particular reason, would you mind adding a comment to that effect?